### ISSUE TITLE
Steppenwolf 6-3 softlocks in multiple places

### ISSUE THREAD

**Issue by Toad06** [ruffle-rs/ruffle#1449]
https://www.y8.com/games/steppenwolf_chapter_6_episode_3

1- At the very beginning of this episode, you cannot check your inventory by pressing the "I" key. If you do so in Flash, then nothing happens; in Ruffle, the game softlocks.

Steps to reproduce: Start a new "Regular game" and press "I".

Removing this part of code prevents the issue from happening (but it also prevents the inventory from being displayed completely on legit situations):
```
tellTarget("_root")
{
	gotoAndStop(/Variables:EtatVide);
}
play();
```

2 (FIXED)- Still at the beginning of the episode, you have to shoot the guard. For some reason, he's immortal in Ruffle.

Steps to reproduce: Start a new "Regular game", look at the meal tray (space), take the soda can, use the soda can (I) on the door, quickly take the gun and shoot the guard.

The following messages are logged in the console:

> Errors: SetTarget failed: _root.MurBalle.MurMechant1 not found
> Target not found: Target="_root.MurBalle.MurMechant1" Base="_level0.Mechant1"

**Comment by Dinnerbone** [comment]
Should be fixed now with one of the commits in #1447 - can you please test and verify?

I tried but I don't have dedicated arrow keys and am not fast enough to shoot the guard 😅

**Comment by Toad06** [comment]
Tried on web and desktop using the nightly build of the day. Unfortunately I'm still able to reproduce both of the issues.

**Comment by Dinnerbone** [comment]
That's strange, I thought at least the first issue would be fixed. I had no issues opening (or not opening) the inventory.

**Comment by Toad06** [comment]
Oh, are you sure you've pressed "I" at the very beginning of the game? I mean you have to do this before pressing the spacebar near the meal tray, as the character will get her tablet back and the inventory will then become accessible again.

**Comment by Dinnerbone** [comment]
Yes! And nothing happened, which I presume was correct?

**Comment by Toad06** [comment]
Absolutely! But then you cannot move the character anymore, you'll need to reload the game.

It seems like the inventory opens without being visible; I'm guessing that because the following messages are logged in the console (they're displayed every time you press "I" on legit situations): 
```
Sound.getVolume: Unimplemented
Unimplemented property _quality
```

**Comment by Toad06** [comment]
Seems like the second issue is not related to `tellTarget`: it's still possible to shoot the guard in Flash, if you remove the following piece of code.
```
tellTarget("_root.MurBalle.MurMechant1")
{
   gotoAndStop(2);
}
play();
```

So it may be an issue with `hitTest` (if so, possibly a duplicate of #1211?).

**Comment by Toad06** [comment]
So it turns out the second softlock is actually the same issue as #1559: `hitTest("targetAsString")` doesn't work but `hitTest(eval("targetAsString"))` does.

**Comment by Herschel** [comment]
Second softlock fixed by #1578

**Comment by Toad06** [comment]
The first softlock no longer occurs as well - assume fixed by #1770.
